source/index/contracts/Index.sol

```diff
@@ -86,6 +86,8 @@ contract Index is Ownable {
     bytes32 locator
   ) external onlyOwner {
 
```

Comment:
put in a comment for this line

// Disallow locator set to 0x0 to ensure list integrity.